lib/Default/AbstractSmrShip.class.php

```diff
@@ -1089,8 +1089,13 @@ protected function doArmourDamage($damage) {
 		return $actualDamage;
 	}
 
+	/**
+	 * Returns the maneuverability rating for this ship.
+	 */
 	public function getMR() : int {
-		return IRound(
+		//700 - [ (ship hit points / 25) + (ship stat factors) ]
+		//Minimum value of 0 because negative values cause issues with calculations calling this routine
+		return Max(0, IRound(
```

Comment:
```suggestion
		return max(0, IRound(
```
PHP, for who knows what reason, is case-insensitive. However, I prefer to use the "canonical" capitalization, which in this case is lowercase.